_format/1.1/index.md

```diff
@@ -1781,6 +1822,183 @@ parameter from this specification, it **MUST** return `400 Bad Request`.
+
+### <a href="#profile-keywords" id="profile-keywords" class="headerlink"></a> Profile Keywords
+
+A profile **SHOULD** explicitly declare "keywords" for any elements that it
+introduces to the document structure. If a profile does not explicitly declare a
+keyword for an element, then the name of the element itself is considered to be its
+keyword. Keywords **MAY** be aliased in any representation through the use of
+[profile descriptors](profile-descriptors), as described below.
+
+All profile keywords **MUST** meet this specification's requirements for [member names].
+
+### <a href="#profile-media-type-parameter" id="profile-media-type-parameter" class="headerlink"></a> `profile` Media Type Parameter
+
+The `profile` media type parameter is used to describe the application of
+one or more profiles to the JSON API media type. The value of the `profile`
+parameter **MUST** equal a space-separated (U+0020 SPACE, " ") list of profile URIs.
+
+> Note: When serializing the `profile` media type parameter, the HTTP 
+> specification requires that its value be surrounded by quotation marks 
+> (U+0022 QUOTATION MARK, "\"") if it contains more than one URI.
+
+A client **MAY** use the `profile` media type parameter in conjunction with the
+JSON API media type in an `Accept` header to _request_, but not _require_, that
+the server apply one or more profiles to the response document. When such a 
+request is received, a server **SHOULD** attempt to apply the requested profiles 
+to its response.
+
+For example, in the following request, the client asks that the server apply the
+`http://jsonapi.org/extensions/last-modified` profile if it is able to.
+
+```http
+Accept: application/vnd.api+json;profile="http://jsonapi.org/extensions/last-modified", application/vnd.api+json
+```
+
+> Note: The second instance of the JSON API media type in the example above is
+  required under the [client's content negotiation responsibilities](#content-negotiation-clients).
+  It is used to support old servers that don't understand the profile parameter.
+
+Servers **MAY** add profiles to a JSON API document even if the client has not
+requested them. The recipient of a document **MUST** ignore any profile extensions 
+in that document that it does not understand. The only exception to this is profiles
+whose support is required using the `profile` query parameter, as described below.
+
+#### <a href="#profiles-sending" id="profiles-sending" class="headerlink"></a> Sending Profiled Documents
+
+Clients and servers **MUST** include the `profile` media type parameter in
+conjunction with the JSON API media type in a `Content-Type` header to indicate
+that they have applied one or more profiles to a JSON API document.
+
+Likewise, clients and servers applying profiles to a JSON API document **MUST** 
+include a [top-level][top level] [`links` object][links] with a `profile` key, 
+and that `profile` key **MUST** include a [link] to the URI of each profile 
```

Comment:
When no profiles are specified in the request document, but they _are_  specified in the `Content-Type` request header or the `profile` URL query parameter, should a `400` error response be sent?

IOW: are JSON API documents in request bodies required to match the `Content-Type` and/or `profile` URL query argument?

Reply:
On second thought, I don't think this makes sense, because it's already valid to just send
```
{
  "data": { … }
}
```

Because http://jsonapi.org/format/#document-top-level already says that only one of `data`, `errors` and `meta` is a MUST-have.

Reply:
> When no profiles are specified in the request document, but they are specified in the Content-Type request header or the profile URL query parameter, should a 400 error response be sent?

This is a good point. I think we want to say that, on requests, if the profiles listed in `Content-Type` don't match those listed in the document body, the server should return a 400. 

> On second thought, I don't think this makes sense, because it's already valid to just send

I don't think that prevents us from requiring a 400 in this case, because none of the requests that are valid today will contain `profile` in the `Content-Type`.